cogs/commands/birthday.py

```diff
@@ -4,10 +4,12 @@
 from discord.ext import commands
 from discord.ext.commands import Bot, Context
 from pytz import timezone, utc
+from sqlalchemy.sql import func
```

Comment:
since this is only for func.count, can we do this without importing a whole new lib? nws if not